src/librbd/librbd.cc

```diff
@@ -519,6 +520,25 @@ namespace librbd {
     return 0;
   }
 
+  int RBD::features_to_string(uint64_t features, std::string *str_features)
+  {
+    std::stringstream err;
+    *str_features = librbd::rbd_features_to_string(features, &err);
+    if (err.str().find("ignoring unknown feature mask") !=std::string::npos)
```

Comment:
Nit: you know if any error is printed in `err` that there was an issue decoding it -- so no need to search for a magic string.

Reply:
hi dillaman， would I  follow the changes below? thanks a lot
if (err.str().size()){
 return -EINVAL;
}

Reply:
Preferably `if (!err.str().empty()) ...`, but yes.

Reply:
I will change it, thanks.